content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -258,26 +258,47 @@ created, and deleted with the `kubeadm token` command. See the
 ### Installing a Pod network add-on {#pod-network}
 
 {{< caution >}}
-This section contains important information about installation and deployment order. Read it carefully before proceeding.
-{{< /caution >}}
-
-You must install a Pod network add-on so that your Pods can communicate with
-each other.
+This section contains important information about networking setup and
+deployment order.
+Read all of this advice carefully before proceeding.
+
+**You must deploy a Pod network add-on so that your Pods can communicate
+with each other.  
+Cluster DNS (CoreDNS) will not start up before a network
+is installed.**
+
+- kubeadm only supports {{< glossary_tooltip text="Container Network Interface" term_id="cni" >}}
+  (CNI) based Pod networks.  
+  kubeadm does not support kubenet.
```

Comment:
i will combine this with the sentence above:

```
You must deploy a {{< glossary_tooltip text="Container Network Interface" term_id="cni" >}}
(CNI) based Pod network add-on so that your Pods can communicate with each other.
```

i don't think we need to mention kubenet.